src/js/stores/DCOSStore.js

```diff
@@ -26,50 +27,42 @@ class DCOSStore extends EventEmitter {
 
     this.data = {
       marathon: new ServiceTree(),
-      mesos: {
-        frameworks: []
-      },
+      mesos: new SummaryList(),
       dataProcessed: false
     };
 
     this.proxyListeners = [
```

Comment:
we recently refactored `StoreMixin` so that it doesn't only work on components. it can work on straight up objects now. cc @MatApple 

we should create a jira for using it here